package/devel/gdb/patches/140-sgidefs.patch

```diff
@@ -0,0 +1,11 @@
+--- a/gdb/mips-linux-nat.c
++++ b/gdb/mips-linux-nat.c
+@@ -31,7 +31,7 @@
+ #include "gdb_proc_service.h"
+ #include "gregset.h"
+ 
+-#include <sgidefs.h>
++#include <asm/sgidefs.h>
```

Comment:
It would be nice to get this into upstream gdb.

Reply:
Doubt they will accept. The GNU sgidefs.h undefines a bunch of stuff from asm/sgidefs.